include/bluetooth/audio/audio.h

```diff
@@ -200,15 +200,21 @@ struct bt_codec_data {
 		.meta = _meta, \
 	}
 
-/* TODO: Remove base once LTV types are defined, are these specific to LC3? */
-#define BT_CODEC_META_BASE               0x01
-
-#define BT_CODEC_META_PREFER_CONTEXT     (BT_CODEC_META_BASE)
-#define BT_CODEC_META_CONTEXT            (BT_CODEC_META_BASE + 1)
+#define BT_CODEC_META_PREFER_CONTEXT     (0x01)
+#define BT_CODEC_META_CONTEXT            (0x02)
+#define BT_CODEC_META_PROGRAM_INFO       (0x03)
+#define BT_CODEC_META_LANGUAGE           (0x04)
+#define BT_CODEC_META_CCID_LIST          (0x05)
+#define BT_CODEC_META_PARENTAL_RATING    (0x06)
+#define BT_CODEC_META_PROGRAM_INFO_URI   (0x07)
+#define BT_CODEC_META_EXTENDED_METADATA  (0xFE)
+#define BT_CODEC_META_VENDOR_SPECIFIC    (0xFF)
```

Comment:
Enums for sure - I made this change quite a while back - before some of the codec defines were changed into enums. Thanks for pointing it out.
I'll add the reference - it would be nice with all the Generic Audio assigned numbers in a single group.